ipcontroller_config.py

```diff
@@ -39,12 +39,12 @@
 # c.IPControllerApp.log_level = 30
 
 # Set the working dir for the process.
-# c.IPControllerApp.work_dir = u'/home/cekees'
+# c.IPControllerApp.work_dir = u'/work1/home/cekees'
 
 # ssh url for engines to use when connecting to the Controller processes. It
 # should be of the form: [user@]server[:port]. The Controller's listening
 # addresses must be accessible from the ssh server
-# c.IPControllerApp.engine_ssh_server = u''
```

Comment:
This is what I added, which seemed to work on one small cluster, but I suspect it's not the right thing to do.